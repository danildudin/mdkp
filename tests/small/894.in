4 3
100 41 10 41 
72 51 5 43 328
64 10 20 91 523
23 7 22 27 40
