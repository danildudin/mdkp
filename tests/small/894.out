141
2
0 1 
